src/main/java/uk/nhs/cactus/common/security/JWTHandler.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.cactus.common.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import java.time.Clock;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class JWTHandler {
+
+  private final Clock clock;
```

Comment:
is the clock actually being used? If not can we remove it or if so should we add the clock to this package as well so that individual services don't need to register a bean for it?

Reply:
I've now added the token generation methods from the EMS, which do use the clock.